packages/web3-core/src/types.ts

```diff
@@ -46,6 +46,8 @@ export interface Web3ConfigOptions {
 	transactionBlockTimeout: number;
 	transactionConfirmationBlocks: number;
 	transactionPollingTimeout: number;
+	transactionReceiptPollingInterval: number;
```

Comment:
This was called `transactionPollingTimeout` in `1.x`, but I believe this is a better name as it's more specific to what it affects - this is a breaking change though

Reply:
@nazarhussain  Yea we probably should, we'd just need the getter/setter of one call the other so they never become out of sync

Reply:
For clarification, I reintroduced `transactionPollingInterval` and have it update both `transactionReceiptPollingInterval` and `transactionConfirmationPollingInterval` if changed